code/modules/mob/dead/observer/orbit.dm

```diff
@@ -86,3 +96,16 @@
 /datum/orbit_menu/ui_assets()
 	. = ..() || list()
 	. += get_asset_datum(/datum/asset/simple/orbit)
+	. += get_asset_datum(/datum/asset/spritesheet/job_icons)
+
+/datum/asset/spritesheet/job_icons
+	name = "job-icon"
+
+/datum/asset/spritesheet/job_icons/register()
+	var/icon/I = icon('icons/mob/hud.dmi')
+	// Get the antag hud part
```

Comment:
blatant player panel thievery comment, you're not slick :trollface:

Reply:
this probably came from emojis in the first place :heck:

Reply:
I also wrote emojis >:(

Reply:
emojis have been in the game since I've been here